src/store/state/state-model.ts

```diff
@@ -177,7 +177,7 @@ export const stateModel: StateModelIF = {
       contactPoint: {
         email: '',
         phone: '',
-        extension: ','
```

Comment:
???

Um, yeah.

Reply:
haha yea, i had to look into this, i thought perhaps it was intentional at first.